### ISSUE TITLE
Became a addon

### ISSUE THREAD

**Issue by felixfong227** [SaltyPW/SaltyPW.github.io#1]
If you can make it became a Chrome/Firefox addons,and i'll be awesome :)

**Comment by MadMarx** [comment]
Yep, that's certainly in the plans although I can't promise any ETA.

**Comment by felixfong227** [comment]
Cool,if you make is as a add on,it will make every thing event faster then every,but for now,is awesome too
